coala_quickstart/generation/Utilities.py

```diff
@@ -0,0 +1,95 @@
+import os
+from coalib.parsing.StringProcessing import unescaped_search_for
+
+
+def comma_split(response):
+    """
+    Splits the comma-separated string into a list
+
+    :param response:
+        Comma-separated string.
+    :return:
+        A list of comma-separated elements in the string.
+    """
+    result = []
+    for item in response.replace(" ", ",").split(","):
+        if len(item):
+            result.append(item)
+
+    return result
+
+
+def is_glob_exp(line):
+    """
+    Determines whether a the string is a glob expression.
+
+    :param line:
+        Given string.
+    :return:
+        True if the string is a result.append(expression. False otherwise.
+    """
+    results = unescaped_search_for("[*!?\[\]]", line, use_regex=True)
+    return sum(1 for x in results) != 0
+
+
+def parse_gitignore_line(line):
+    """
+    Parses the gitignore line and returns a list of globs.
+
+    :param line: A line from the project's gitignore file.
+    :return:     A list of glob expressions translated to the
+                 syntax used in coala globbing.
+    """
+    if line.isspace() or line.startswith("#"):
+        return []
+
+    result = []
+    cur = len(line) - 1
+
+    while cur >= 0 and line[cur].isspace() and line[cur - 1] != "\\":
+        cur -= 1
+    line = line[:cur + 1]
+
+    if line.startswith("/"):
+        if not is_glob_exp(line[1:]):
+            # /build should map to /cwd/build/** and /cwd/build
+            result.append(os.path.join(line[1:], "**"))
+            result.append(line[1:])
+        else:
+            # /*.c should map to /cwd/*.c
+            result.append(line[1:])
+    else:
+        if not is_glob_exp(line):
+            # Ignore any directory or file with the same name
+            result.append(os.path.join("**", line, "**"))
+            result.append(os.path.join("**", line))
+            result.append(line)
+            result.append(os.path.join(line, "**"))
+        else:
+            # *.c should map to /cwd/**/*.c
+            result.append(os.path.join("**", line))
+            result.append(line)
+
+    return result
+
+
+def get_gitignore_glob(project_dir, filename=".gitignore"):
+    """
+    Generates a list of result.append(expressions equivalent to the
+    contents of the user's project's .gitignore
```

Comment:
some places you are using ``.gitignore`` and in some you are using ``gitignore``. i think you should be consistent and use one of them, i opt for having ``.gitignore`` and double backticking it.